src/extension.ts

```diff
@@ -268,57 +255,98 @@ export async function activate(
     );
   });
 
-  const inputReloadDomains = [
-    "input_boolean",
-    "input_datetime",
-    "input_number",
-    "input_select",
-    "input_text",
-  ];
+  context.subscriptions.push(
+    vscode.commands.registerCommand(`${extensionId}.inputReload`, async (_) => {
+      await Promise.all(
+        inputReloadDomains.map(async (domain) => {
+          await client.sendRequest("callService", {
+            domain,
+            service: "reload",
+          });
+        })
+      );
+      await vscode.window.showInformationMessage(
+        "Home Assistant inputs reload called!"
+      );
+    })
+  );
 
   context.subscriptions.push(
     vscode.commands.registerCommand(
-      "vscode-home-assistant.inputReload",
-      async (_) => {
-        await Promise.all(
-          inputReloadDomains.map(async (domain) => {
-            await client.sendRequest("callService", {
-              domain,
-              service: "reload",
-            });
-          })
-        );
-        await vscode.window.showInformationMessage(
-          "Home Assistant inputs reload called!"
-        );
+      `${extensionId}.homeassistantCheckConfig`,
+      async () => {
+        await client.sendRequest("checkConfig");
       }
     )
   );
+  context.subscriptions.push(
+    vscode.commands.registerCommand(`${extensionId}.getErrorLog`, async () => {
+      await client.sendRequest("getErrorLog");
+    })
+  );
 
   context.subscriptions.push(
     vscode.commands.registerCommand(
-      "vscode-home-assistant.homeassistantCheckConfig",
+      `${extensionId}.renderTemplate`,
       async () => {
-        await client.sendRequest("checkConfig");
+        const editor = vscode.window.activeTextEditor;
+        const selectedText = editor.document.getText(editor.selection);
+        await client.sendRequest("renderTemplate", { template: selectedText });
       }
     )
   );
+
   context.subscriptions.push(
     vscode.commands.registerCommand(
-      "vscode-home-assistant.getErrorLog",
+      `${extensionId}.fetchEntities`,
       async () => {
-        await client.sendRequest("getErrorLog");
+        await client.sendRequest("fetchEntities");
       }
     )
   );
 
   context.subscriptions.push(
     vscode.commands.registerCommand(
-      "vscode-home-assistant.renderTemplate",
-      async () => {
-        const editor = vscode.window.activeTextEditor;
-        const selectedText = editor.document.getText(editor.selection);
-        await client.sendRequest("renderTemplate", { template: selectedText });
+      `${extensionId}.openEntityLogbook`,
+      (treeItem) => {
+        const { hostUrl } = vscode.workspace.getConfiguration()[extensionId];
```

Comment:
Need a clever way to detect if we're using `process.env.HASS_SERVER` and `process.env.HASS_TOKEN` that's outside language server.